### PR TITLE
Typo in Step 43 of Building a Cipher #54837

### PR DESCRIPTION
**Pull request by adi-rajput**
Checklist:

[x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org/).
[x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
[x] My pull request targets the main branch of freeCodeCamp.
[x] I have tested these changes either locally on my machine, or GitPod.
Closes ##54837
Screenshot 2024-05-16 231738

### CODE REVIEW

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Removing the checklist in the PR template makes so that we can't accept your PR

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-js-fundamentals-by-building-a-gradebook-app/6626b8dcf5057f896f948440.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-4
 
 Now that the teacher has all of the information they need, they want to be able to message the student with the results. 
 
-Complete the `studentMsg` function with `totalScores` and `studentScore` for parameters. The function should return a string representing a message to the student.
+Complete the `studentMsg` function with `totalScores` and `studentScore` for parameters. The function should return string representing a message to the student.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-js-fundamentals-by-building-a-gradebook-app/6626b8dcf5057f896f948440.md:12]: this change should not be incldued in this PR

Comment (adi-rajput) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-js-fundamentals-by-building-a-gradebook-app/6626b8dcf5057f896f948440.md:12]: So what  should I do ?

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553995f412dd8122ed38e4a.md
Comment (adi-rajput) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553995f412dd8122ed38e4a.md:1]: Only requested file is changed , is it okay?

Comment (ilenia-magoni) [reply curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553995f412dd8122ed38e4a.md:1]: no, you deleted the file, it needs to be unchanged from the main branch

**Pull request closed without merging**